src/get-started/java-binding.md

```diff
@@ -1062,8 +925,8 @@ In case the service build definition needs to be configured, ensure that
 the following required metadata is present in the service artifact JAR:
 
 - Entries in the JAR manifest:
-    - "Plugin-Id": must be set to "groupId:artifactId:version", e.g.,
-    `com.exonum.example.timestamping:timestamping-demo:1.0.2`.
+    - "Plugin-Id": must be set to "1:groupId/artifactId:version", e.g.,
+    `1:com.exonum.example.timestamping/timestamping-demo:1.0.2`.
     - "Plugin-Version": must be set to the project version, e.g., `1.0.2`.
```

Comment:
Should the version be equal to a version in the plugin-id? maybe describe why such duplication is needed?

Reply:
Yes, it must. I'd rather not to: there is a technical reason for that that is none of our users concern.